### PR TITLE
Updated Doc links and added website link to hello there window

### LINKED ISSUE #207: Redirect to pixieditor.net instead of GitHub (in menu items)
Currently menu items redirecting to website, are redirecting to GitHub instead of pixieditor.net

### PR DESCRIPTION
**Pull request by CPKreuz**
Closes #207

### CODE REVIEW

File: PixiEditor/Views/Dialogs/HelloTherePopup.xaml [hunk 1]
```diff
@@ -220,6 +220,19 @@
                 </StackPanel>
 
                 <uc:AlignableWrapPanel Grid.Row="3" HorizontalContentAlignment="Center" HorizontalAlignment="Center" Margin="0,5,0,15">
+                    <Button Command="{Binding OpenHyperlinkCommand}" CommandParameter="https://github.com/PixiEditor/PixiEditor"
```
Comment (flabbet) [inline PixiEditor/Views/Dialogs/HelloTherePopup.xaml:223]: Can you tell me how did you test that, so you didn't notice this xD?

**Review by flabbet — CHANGES_REQUESTED** [review]
Too much buttons, can you replace them with nice smaller logos?

File: PixiEditor/Views/Dialogs/HelloTherePopup.xaml (hunk 1 above)
Comment (CPKreuz) [reply PixiEditor/Views/Dialogs/HelloTherePopup.xaml:223]: uhm...

**Review by flabbet — CHANGES_REQUESTED** [review]
Use this for donate https://feathericons.com/?query=money icon, also make the buttons use hand cursor on hover

**Review by flabbet — APPROVED** [review]

**Pull request merged**